cmd/upgradehelper/cmd/root.go

```diff
@@ -35,9 +38,9 @@ func init() {
 	debug := envGetBool("DEBUG", false)
 	trace := envGetBool("TRACE", false)
 
-	rootCmd.PersistentFlags().BoolVar(&logDebug, "debug", debug, "set logging level to debug")
-	rootCmd.PersistentFlags().BoolVar(&logTrace, "trace", trace, "set logging level to trace")
+	RootCmd.PersistentFlags().BoolVar(&logDebug, "debug", debug, "set logging level to debug")
+	RootCmd.PersistentFlags().BoolVar(&logTrace, "trace", trace, "set logging level to trace")
 
-	rootCmd.Flags().StringVar(&kubeConfigPath, "kubeconfig", os.Getenv("KUBECONFIG"), "Path to the kubeconfig file")
-	rootCmd.Flags().StringVar(&kubeContext, "kubecontext", os.Getenv("KUBECONTEXT"), "Context name")
+	RootCmd.Flags().StringVar(&KubeConfigPath, "kubeconfig", os.Getenv("KUBECONFIG"), "Path to the kubeconfig file")
+	RootCmd.Flags().StringVar(&KubeContext, "kubecontext", os.Getenv("KUBECONTEXT"), "Context name")
```

Comment:
They should've been the persistent flags. Thanks for pointing out.

```
./upgrade-helper --kubeconfig=/path/to/kubeconfig --kubecontext=test-context version-guard hvst-upgrade-xxxxx
```

The two subcommands will leverage them for the location of the desired kubeconfig file and the context (optional). We don't explicitly use the two flags because we inject the value via environment variables. They're convenient for us when doing the development and testing.